app/assets/javascript/dropdown_select_all.js

```diff
@@ -0,0 +1,40 @@
+(function($) {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
```

Comment:
Ah I wasn't aware of this. I've changed the module to conform to this standard, which should also make it easier if we want to move this to the admin template later.